VideoLocker/src/test/java/org/edx/mobile/view/CourseUnitNavigationActivityTest.java

```diff
@@ -520,15 +531,21 @@ public void testUnitFragmentCreation() {
         @Parameterized.Parameter(value = 1)
         public Class expected;
```

Comment:
Instead of just `input` and `expected`, you should add some description to these variable names as well.